src/main/java/core/basesyntax/transaction/TransactionHandler.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.transaction;
+
+import core.basesyntax.model.Fruit;
+
+public interface TransactionHandler {
```

Comment:
you should have 4 implementations for this interface:
balance
supply
return
purchase